weird-flex.html

```diff
@@ -25,7 +25,7 @@
   <body>
     <div>
       <span id="output">weird flex but ok</span>
-      <button id="new">🔄</button>
+      <button id="new">▶️</button>
```

Comment:
Why? I used 🔄 to mean "regenerate"; ▶️ implies something along the lines of "continue"